### DISCUSSION TITLE
Call error: echo Error: GAMSCMEX did not write a next script.

### DISCUSSION THREAD

**Discussion by wanyingw193** [NREL/ReEDS-2.0#64] [Q&A: Execution]
Hi,

When calling a case scenario with call_casename_2_ercot_seq.cmd, ReEDS stops at one year and gamsnext.cmd shows the error as:
------------
@echo off
echo Error: GAMSCMEX did not write a next script.

exit 6
----------
How could this be handled? Thanks!

**Comment by vcelloho** [comment]
It looks like the needed file didn't generate. Have you run into this issue before in prior runs?

Gams documentation mentions that exit 6 is a parameter error return code. I'm looking into more ifnormation.

**Reply by wanyingw193** [reply]
Thank you! I found out the cause was that I accidentally deleted a letter, but after I fixed that, the model was running again. Thanks! One strange thing is that the gamslog.txt and .lst files don't show this error.

For my most recent run, the model stopped in B_osprey.gms after a few years of running, e.g. the model year entered was 2010-2050, but it stopped in 2038. Is it possible that I can still get the output of the model in years 2010-2036? Or do I have to run it again to get the results? Thank you so much!
